dandi/files.py

```diff
@@ -508,7 +508,7 @@ def get_validation_errors(
         schema_version: Optional[str] = None,
         devel_debug: bool = False,
     ) -> List[str]:
-        errors: List[str] = pynwb_validate(self.filepath, devel_debug=devel_debug)
```

Comment:
Note that `pynwb.validate` comes for free with the NWBInspector: https://github.com/NeurodataWithoutBorders/nwbinspector/blob/dev/nwbinspector/nwbinspector.py#L324-L333

However, I noticed your `pynwb_validate` helper does some additional filtering on top of this, so we may have to update our end to ensure it behaves 100% the same way.

Reply:
I wonder if instead of replacing "our" invocation of nwb's stock `.validate` we could just instruct inspector to skip nwb stock validation?

Reply:
Would be better for everyone else to user the superior version here, no? The inspector validation is only the vanilla type right now, but we should improve it by porting over your modifications.

Reply:
Never mind, Ben walked me through what this issue is about. Looks like a PR on PyNWB is in progress: https://github.com/NeurodataWithoutBorders/pynwb/pull/1432

When it is, maybe we should update the functionality so the ignoring of 2.10 is no longer needed within `dandi`, and the `validate` can be one of the burdens of the Inspector.

For now, we will add a flag to the Inspector to skip validation so we can run yours instead.

Reply:
;-) superior is good.  I guess we would just need to encode (if we haven't yet) desired assumptions on nwb validation in our tests to guarantee that nwbinspector refactorings do not make them go away etc.

Reply:
please digest it for me a bit more @CodyCBakerPhD -- will we need to wait for new pynwb release?